### ISSUE TITLE
Portrait Rotation issue on Windows 8 Surface

### ISSUE THREAD

**Issue by dellis1972** [mono/MonoGame#1269]
I got this report from a user
"I recently ran into another issue that might be related to the resolution issue with the Surface.  When I run on a monitor that has been rotated into portrait mode the graphics device settings look like they're based on the unrotated values.  The resulting image ends up getting stretched to/squashed to fit the rotated aspect ratio."

This is the test app which demos the problem,

https://dl.dropbox.com/u/18881050/bug.zip

To show the problem you can either run it on a monitor that has been rotated into portrait mode or run it on the Win8 Surface simulator with that rotated.  NOTE that you need to do the rotation before starting the app.

**Comment by tomspilman** [comment]
Win8 is old... closing.
